app/assets/stylesheets/application/components/header.scss

```diff
@@ -86,18 +87,53 @@ header {
     padding: 3rem 0 4rem;
   }
   .logo {
+    display: none;
+    img {
+      height: auto;
+      max-width: 100px;
+    }
+    @media #{$small-only} {
+      display: inline-block;
+    }
+  }
+
+  .site-name {
+    background: none;
+    @media #{$medium-up} {
+      background: url("http://localhost:3000/assets/sec-logo.png");
```

Comment:
Rails has helpers to generate the url needed here, since it will differ between development and production environments. Try `background: image-url('sec-logo.png')`.

Reply:
Ah, that was very silly of me to leave the full URL in there. Thanks for LMK the correct syntax for rails.